### PR TITLE
Use platform exporter

### PR DESCRIPTION
**Pull request by nonnontrivial**
This PR pivots the repo into using only the `WatsonExporter` from the `@botmock/export` package, thus turning it into a runnable sample project.

### CODE REVIEW

File: .vscode/settings.json [hunk 1]
```diff
@@ -0,0 +1,3 @@
+{
+  "deno.enable": false
```
Comment (obaid) [inline .vscode/settings.json:2]: we should remove `.vscode` from the repo

File: README.md [hunk 2]
```diff
@@ -1,100 +1,29 @@
...
 
 ## Overview
 
-### Botmock project structure
-
-Intents on connectors in your original project are what break up the flow into separate Dialog Nodes in Watson. Each Dialog Node in Watson will have as a condition the intent that is connected to it. Once this intent is recognized in Watson, responses will be made that correspond to the content blocks connected (without intents) to the first content block.
-
-### Caveats
-
-> Note that for data surrounding [slot-filling](https://cloud.ibm.com/docs/services/assistant?topic=assistant-tutorial-slots-complex) in your original project to appear available in the Watson Assistant dashboard, you must navigate to the node containing slot-filling within the **Dialog** section of the skill and click **Customize** and then turn slot-filling on. Upon doing so, you should discover your data from your original project appears automatically within the dashboard.
```
Comment (obaid) [inline README.md:33]: do the caveats not apply in this new exporter?

File: README.md [hunk 3]
```diff
@@ -1,100 +1,29 @@
...
+`./output` should be generated in your project root.
 
-- go to the [IBM dashboard](https://cloud.ibm.com/)
-- click **Services**
-- click on one of the **Services**, or **Create resource** and then **Watson Assistant** and **Create**
-- click **Launch Watson Assistant**
-- click on an existing assistant, or **Create assistant**
-- click **Add dialog skill**
-- click **Import skill**
-- choose the `.json` file generated by this script
```
Comment (obaid) [inline README.md:100]: we should still give them instructions on what to do post export?

File: README.md (hunk 2 above)
Comment (nonnontrivial) [reply README.md:33]: Some things have changed in Watson, but this rule still seems to apply, yes.

File: README.md (hunk 3 above)
Comment (nonnontrivial) [reply README.md]: Yes, agreed 👍.

**Pull request merged**